contrib/python-pybind11/template.py

```diff
@@ -0,0 +1,78 @@
+pkgname = "python-pybind11"
+pkgver = "2.11.1"
+pkgrel = 0
+build_style = "python_pep517"
+make_build_args = ["--skip-dependency-check"]
+hostmakedepends = [
+    "cmake",
+    "ninja",
+    "pkgconf",
+]
+makedepends = [
+    "boost-devel",
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+    "python-wheel",
+]
+checkdepends = ["python-pytest"]
+pkgdesc = "Seamless operability between C++11 and Python"
+maintainer = "Duncan Bellamy <dunk@denkimushi.com>"
+license = "BSD-3-Clause"
+url = "https://pybind11.readthedocs.io/en/stable/index.html"
+source = f"https://github.com/pybind/pybind11/archive/refs/tags/v{pkgver}.zip"
+sha256 = "b011a730c8845bfc265f0f81ee4e5e9e1d354df390836d2a25880e123d021f89"
+options = ["!cross"]
+
+
+def post_configure(self):
+    from cbuild.util import cmake
+
+    cmake.configure(
+        self,
+        build_dir="build-tests",
+    )
+
+
+def post_build(self):
+    from cbuild.util import cmake
+
+    cmake.build(self, "build-tests")
+
+
+def do_check(self):
+    # deselected test has RuntimeError
+    self.do(
+        "sh",
+        "-c",
+        "cd build-tests/tests && python3 -m \
+        pytest --deselect test_exceptions.py::test_cross_module_exception_translator",
+    )
+
+
+def post_install(self):
+    import sys
+
+    _pypath = f"/usr/lib/python{sys.version_info[0]}.{sys.version_info[1]}/site-packages"
```

Comment:
can't use sys.version_info because that is using the host python and not the one actually being built against in cports

not sure what would work in this case though

Reply:
I thought the cbuild was done in some kind of container or chroot?

could install to temp folder and “find” the path?

Reply:
could also move the whole building tests into `init_check` so no longer !cross ?

Reply:
it is in a bwrap container but the `def:` hooks are evaluated on the host with the python used to run cbuild and not the python that runs `python -m build`, etc

Reply:
> could also move the whole building tests into init_check so no longer !cross ?
and yeah that sounds good

Reply:
changed it, find is finding /usr/include first though?

Reply:
/usr/include sorts before /usr/lib lexicographically, you could find in self.destdir / usr/lib

Reply:
that works, but stripping destdir still ends up stripping destdir / usr/lib/python

so stripped string starts with python version number